clang/include/clang/Driver/Options.td

```diff
@@ -2841,6 +2841,10 @@ def fcf_protection : Flag<["-"], "fcf-protection">, Group<f_Group>,
   Visibility<[ClangOption, CLOption, CC1Option]>,
   Alias<fcf_protection_EQ>, AliasArgs<["full"]>,
   HelpText<"Enable cf-protection in 'full' mode">;
+def fcf_branch_label_scheme_EQ : Joined<["-"], "fcf-branch-label-scheme=">,
```

Comment:
```suggestion
def mcf_branch_label_scheme_EQ : Joined<["-"], "mcf-branch-label-scheme=">,
```

Use `m` option rather than `f` option since I assume this should apply to RISC-V only.

Reply:
Fixed